### DISCUSSION TITLE
Automate Fan Control on Boot

### DISCUSSION THREAD

**Discussion by sherbmeister** [Rem0o/FanControl.Releases#1496] [Q&A]
Hi,
I'm on windows 11 and every time I boot up, my fans are on full speed I'd love it if someone can help me out with how to set Fan Control to automatically do "Pair controls and speeds" without me manually going into it and pressing the buttons.

Thanks!

**Comment by Alekak1** [comment]
Hi,
I have the same problem. When starting PC, Fan Control won’t turn on automatically and the case fans goes 100%. When I enter the application:
1)	Before last update: My case fan control had information that other program is controlling the fan. I had to disable and enable it (inside FC) to work again.
2)	After update (V171): there is a pop out “fan control was running under another instance [SID]”. I have checked the SID and it is my mail user. However this time I didn’t have to do anything and fans goes back to the curve.
I have MSI afterburner and GCC from gigabyte, and thought they case the problem, but after turning them off problem still exists. I have also read the issue #1053 but turning off the PC does not solve that problem.

I would greatly appreciate your help with this problem :)
